src/Controllers/ContentController.php

```diff
@@ -2225,4 +2258,122 @@ public function report(Request $request)
+    {
+        $types = $request->get('included_types');
+        $artist = $request->get('artist');
+        $brand = $request->get('brand', config('railcontent.brand', ''));
+        if ($request->has('count_filter_items')) {
+            ContentRepository::$countFilterOptionItems = $request->has('count_filter_items');
+        }
+        $catalogMetaAllowableFilters = ContentRepository::$catalogMetaAllowableFilters;
+        if (in_array('shows', $types)) {
+            $types =
+                array_merge($types, array_values(config('railcontent.showTypes')[config('railcontent.brand')] ?? []));
+        }
+        foreach ($types as $type) {
+            $type = $this->getContentTypeForMetaData($type);
+            if (array_key_exists($type, config('railcontent.cataloguesMetadata.' . $brand))) {
+                $catalogMetaAllowableFilters = config('railcontent.cataloguesMetadata.' . $brand . '.' . $type . '.allowableFiltersMobile', ContentRepository::$catalogMetaAllowableFilters);
+            }
+        }
+        if (count($types) > 1) {
+            $catalogMetaAllowableFilters = config('railcontent.cataloguesMetadata.' . $brand . '.all.allowableFiltersMobile', ContentRepository::$catalogMetaAllowableFilters);
+        }
+        ContentRepository::$catalogMetaAllowableFilters = $catalogMetaAllowableFilters;
+
+        $requiredFields = $request->get('required_fields', []);
+        $requiredFields = array_merge($requiredFields, ['artist,' . $artist]);
+        if ($request->has('title')) {
+            $requiredFields = array_merge($requiredFields, ['title,%'.$request->get('title').'%,string,like']);
+        }
+
+        $lessons = $this->contentService->getFiltered( $request->get('page', 1),
+                                                       $request->get('limit', 12),
+                                                       $request->get('sort', '-popularity'),
+                                                       $types,
+                                                       $request->get('slug_hierarchy', []),
+                                                       $request->get('required_parent_ids', []),
+                                                       $requiredFields,
+                                                       $request->get('included_fields', []),
+                                                       $request->get('required_user_states', []),
+                                                       $request->get('included_user_states', []),
+                                                       $request->get('with_filters', true));
+        $totalPlays = $this->userContentProgressService->countByArtistTypesUserProgress(
+            ['song'],
+            $artist
+        );
+
+        $content = new ContentEntity();
+        $content['name'] = $artist;
+        $content['type'] = 'artist';
+        $content['thumbnail_url'] = 'https://dpwjbsxqtam5n.cloudfront.net/shows/challenges.jpg';
```

Comment:
I assume this is a temporary placeholder, but just want to confirm it was intentional and not something that was missed.

Reply:
Ah good catch. I replaced with the default image for the moment. Thanks!